SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
 
 You must generate schema when you add or change any [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) that you want to deploy to SpatialOS.
 
+> Whenever you generate schema, the GDK automatically creates schema for classes with [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) or [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs). If a class does not have replicated properties or RPCs, then the GDK does not generate schema for it.
```

Comment:
```suggestion
> Whenever you generate schema, the GDK creates schema for you; for classes with [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) or [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs). It stores shema in your project's SchemaDatabase file. If a class does not have replicated properties or RPCs, then the GDK does not generate schema for it.
```

Adding this reference the mention of schema database below, at the bottom. Should the file name be in backticks as it's a file name?

Also, do we need "automatically" now?

Reply:
I reckon "automatically" does the same job as "for you", but happy to make the change.